packages/split-button/docs/SplitButton.stories.tsx

```diff
@@ -193,7 +181,7 @@ const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
       </StoryWrapper>
 
       {IS_CHROMATIC && (
```

Comment:
You know what, since it looks like we only wanted it in Chromatic, and now we're saying to ignore it, you might as well just plain remove it.

I thought it was still going to show in Storybook, but I guess it didn't have that benefit.

Reply:
I was thinking this too - I can remove it